src/main/java/com/redhat/labs/lodestar/resource/ArtifactResource.java

```diff
@@ -0,0 +1,91 @@
+package com.redhat.labs.lodestar.resource;
+
+import java.util.List;
+import java.util.Optional;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+import javax.ws.rs.BeanParam;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+import com.redhat.labs.lodestar.model.Artifact;
+import com.redhat.labs.lodestar.model.ArtifactCount;
+import com.redhat.labs.lodestar.model.GetListOptions;
+import com.redhat.labs.lodestar.model.GetOptions;
+import com.redhat.labs.lodestar.service.ArtifactService;
+
+@Path("/api/artifacts")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+@Tag(name = "Artifacts", description = "Artifact API")
+public class ArtifactResource {
+
+	@Inject
+	ArtifactService service;
+
+	@POST
```

Comment:
if you accept other feedback I gave (update only) this would become a `put` I guess

Reply:
changed to PUT